rtsp.go

```diff
@@ -176,6 +177,13 @@ func (rc *rtspCamera) clientReconnectBackgroundWorker(codecInfo videoCodec) {
 					rc.logger.Infof("reconnected to rtsp server url: %s", rc.u)
 				}
 			}
+
+			// TODO(hexbabe): Delete when discovery API is available.
```

Comment:
what do we need to do to actually serve the discover endpoint from a module?

Reply:
This ticket https://viam.atlassian.net/browse/RSDK-8745

Reply:
Oh shoot I read that wrong.

What we need to do is create a new Go struct to send as a generic interface{} message that makes sense for discovery (that's very vague but basically similar to what we discussed at the AV eng focus meeting). Then we can let the mobile app honor the same struct spec to unpack the response message on the frontend.